Makefile

```diff
@@ -27,7 +27,7 @@ BUILD := ./build
 TOOLS_CMD_ROOT=./cmd/tools
 INTEG_TEST_ROOT=./host
 INTEG_TEST_DIR=host
-INTEG_TEST_XDC_ROOT=./hostxdc
+INTEG_TEST_XDC_ROOT=./host/hostxdc
```

Comment:
I think /host/xdc looks better than /host/hostxdc. What do you think?

Reply:
updated